contracts/dezswap_pair/src/contract.rs

```diff
@@ -625,12 +627,16 @@ fn compute_offer_amount(
 
     let one_minus_commission = Decimal256::one() - commission_rate;
     let inv_one_minus_commission = Decimal256::one() / one_minus_commission;
+    let before_commission_deduction: Uint256 = ask_amount * inv_one_minus_commission;
```

Comment:
I guess this line also has decimal loss which needs ceiling..?

Reply:
https://github.com/dezswap/dezswap-contracts/pull/5/commits/b29775c7ae482f6b304bc3e011776a928409f836